src/deskstar-frontend/pages/userRequests.tsx

```diff
@@ -0,0 +1,61 @@
+import Head from "next/head";
```

Comment:
maybe we should put this page under /users/requests and the the overview under /users

Reply:
True, I will refactor this.